app/controllers/games_controller.rb

```diff
@@ -4,6 +4,14 @@
 class GamesController < ApplicationController
   before_action :set_game, only: %i[show update destroy]
 
+  def render_game_with_state_and_player(game)
+    render json: game.as_json.merge(
+      :game_state => "#{game.board.game_over? ? "GAME OVER" : "IN-PROGRESS"}",
+      :current_player => "#{game.board.current_player}"
+    )
+  end
```

Comment:
This logic should probably be in a view or in the model.

Reply:
👍🏽